### ISSUE TITLE
[Anon] Bug report: Chaos Space Marines - Codex.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#820]
**File:** Chaos Space Marines - Codex.catz

**BattleScribe version:** 1.15.07

**Platform:** Android

**Dropbox:** No

**Description:** Chaos Cultists. Problem adding flamer or heavy stubber to min. squad.

**Comment by NebSeele** [comment]
What version are you using? There should be no issue adding a flamer or heavy stubber to a min squad as long as a Heavy Weapon selection is made. The min/max selection modifiers don't apply till around 20 models so it's an arbitary 1 max until then.

**Comment by cbteom** [comment]
I'm having the same problem. Roster Editor v1.15.07, Warhammer 40,000 7th edition v17, Chaos Space Marines Codex (2012) v137.

When trying to take one squad of cultists with a heavy weapon; since one cultist in a squad of 10 can replace his autopistol with a heavy weapon, I should be able to decrease total number of cultists to eight when I purchase a heavy weapon cultist for an additional 4pts before taking the heavy weapons option. At its current form the best we can do is to get 11 cultists, one with a heavy weapon.

In summary, a 10-man cultist squad with a heavy weapon should cost 55, (50+5 for the weapon) while it is costing 59 now (50+4 for the cultist + 5 for the weapon).

**Comment by NebSeele** [comment]
I slightly reworked the wordings on the entry, and abolished the Heavy weapons actual cultist tax in version 138/139. Heavy weapon's are now Heavy Weapon Upgrade. There should actually be no need to reduce the unit count as it's only the weapon profiles for the weapon selections

In the case of the Heavy Weapons Cultist, I could rework the modifiers again and give him a standalone model entry profile (especially since I am working on Khorne Daemonkin and that ultimately paints a bigger picture). the numbers go off of "Cultists in Unit" anyways so it wouldn't be that hard to fix in comparison to the last issue with the heavy weapons cultist. What does everyone want?

**Comment by amis92** [comment]
Well, I think we should be consistent. I think the "model with upgrade" is now more used, but if CSMs are the other way ("model" + "ugrade" separately), then well, stay that way.

**Comment by NebSeele** [comment]
TL:DR Warning: Some rambling about entry design choice.

For the most part, I try to stay consistent with the other codices where possible. I've generally used the Space Marine file as the base due to its timing with the switch to here being around its release, plus this book and the SM sharing some vague similarities. I've also used the Space Wolves on a couple of times, but more for the formations and the way they implemented it. May check out the rest though (and Necrons current book has also given me an idea entry wise, especially since it's easier resolutions to old fixes, reduces the size and fixes up another issue)

As it is, the cultist file is technically in its third iteration of the entry layout. The first being the Model + Upgrade (Which wasn't reliable as is when I started the PoP and broke once profiles were being built out as the CSM catalogue was very bare minumum as a whole), and the second being the Cultists with the upgrade being purchased at the same time (which collectivity and mix/match issues plagued).

The Heavy Weapons cultist has been a problem with both the first and the second iterations (the second in particular, couldn't account for every numbering sequence that is possible with mixed units, making modifications a nightmare), marking the Hybrid third iteration. 

The Heavy Weapons Cultist's requirements are based on each iterations of 10 models in the unit, so the Units in total was re-introduced and the guns only kept aside (not that they had much else to begin with). This allowed the mix/match while still having a fixed point to measure the Heavy Weapons off of. The only other option is to assign the cultists their choices personally, as the terminator entry does, but for an expendable unit such as cultists, that's a bit... counterproductive to do for so little difference in the end.

Khorne Daemonkin actually has the preliminaries of how I intend to take the entry. While not much different overall, the entry will actually be more complete as a whole (hopefully). 

In both cases, I'll change the Cultist's cost number to incorporate the Heavy Weapons cultist as a separate model and set the numbers at 8/1, 17-18/1 and 26-28/1, allowing for the 10, 20 and 30 cultists (when added up) respectively. Should save some confusion then.

**Comment by cbteom** [comment]
Works perfectly now. Thanks.
